deploy/deploy-with-olm.yaml

```diff
@@ -41,6 +41,17 @@ spec:
   source: ocs-catalogsource
   sourceNamespace: openshift-marketplace
 ---
+# Setting this value in the configmap ensures CSI isn't deployed by client
```

Comment:
Can you pls add this to the go code? This is a generated file.

Reply:
ya, I'm currently doing it, was wondering why there was no cm of ocs for rook is created 🤔, maybe no once is using deploy-with-olm & deploy-manager anymore?

Reply:
This is only for the tests on the local cluster, Ci uses install scripts.

Reply:
so deploy-manager is used to generate deploy-with-olm.yaml, also deploy ocs along w/ dependents (where does make install-* scripts are used), also part of functests.

Reply:
I did not get it, Can you pls rephrase it?

Reply:
I was trying to understand what tasks does deploy-manager carry out.